### ISSUE TITLE
 Have the possibility to sync the password from the nextcloud to the rocketchat to acces to rocketchat without nextcloud

### ISSUE THREAD

**Issue by yodatak** [RocketChat/rocketchat_nextcloud#22]
When we acces to rocketchat via android app , we can't relied on nextcloud authentification so we shoud have a way to sync the nextcloud password with rocketchat or somethink like only using postgresql and make all go on the same user database ;)

**Comment by pierrelocus** [comment]
Thanks for the information, I'll also look at this :)

**Comment by skug67** [comment]
I agree that it's very important for users to be able to access RocketChat w/o going through Nextcloud.  Whether it's sync'ing the passwords or using alternative authentication methods (which RocketChat clearly permits) there ought to be a way to make it happen.

UPDATE -- I see now that one can manually enter a password in RocketChat and still access it from Nextcloud.  So it is possible to use both with the Nextcloud interface and an Android or iOS app.  it would certainly be nice for the process to be automated (including automatic updates to the RocketChat password if the password is changed in Nextcloud).  But given the existence of a manual workaround it's not an urgent issue.

**Comment by Fuseteam** [comment]
wouldn't using nextcloud as an sso provider
also solve this issue?

**Comment by ruvenss** [comment]
> wouldn't using nextcloud as an sso provider also solve this issue?

Totally does, but the idea is to have NextCloud on top and rocket.chat integrated
